.github/SAVED_REPLIES.md

```diff
@@ -0,0 +1,75 @@
+
+While these Saved Replies attempt to give you the ability to quickly reply to Issues and Pull Requests, they are not meant to be the _exact_ response you should use every time.  Consider customizing them to fit the context of the Issue or Pull Request contribution you are replying to.  You will be best served by welcoming the contributor to the project (if its their first issue/PR), thanking them for their contribution, giving them context to your response (especially if you're closing their issue/PR), and noting next steps (e.g., issue milestoned for a specific release, open to them submitting a PR to resolve an issue, sending a PR to someone else to review).
+
+You can add these saved replies to [your personal GitHub account here](https://github.com/settings/replies).
+
+_Sources: [1](https://github.com/angular/angular/blob/master/docs/SAVED_REPLIES.md), [2](https://github.com/angular/angular-cli/blob/master/.github/SAVED_REPLIES.md), [3](https://github.com/prometheus/docs/blob/master/snippets/saved_replies.md), & [4](https://gist.github.com/jywarren/c9a80e0e53f42208974683aa01c623c8)._
+
+## Issue: already fixed (v1)
+```
+Thanks for filing this Issue!  Fortunately it is now obsolete due to changes in a recent release.  Please update to the most recent version to resolve the problem.
+
+If you are still having problems after updating, then please provide additional details for us to try and replicate your issue.
+```
+
+## Issue: don't understand (v1)
+```
+I'm sorry but I don't understand the problem you are reporting.  Would you please rephrase your issue so I can attempt to better understand it?
+```
+
+## Issue: can't reproduce (v1)
+```
+Thanks for filing this Issue!  Unfortunately I cannot reproduce the problem following the instructions you provided.  We require that reported issues have reproduction steps that highlights the problem.
+
+If the problem still exists for you, then please include any additional information on how to reproduce it.
+```
+
+## Issue: behaving as expected (v1)
+```
+It appears this behaves as expected.  If you still feel there is an issue, please provide further details on your problem.
+```
+
+## Issue: template missing (v1)
+```
+Thanks for filing this Issue!  Please note that we have Issue templates for [bug](https://github.com/vxcontrol/soldr/blob/master/.github/ISSUE_TEMPLATE/1-bug-report.md), [enhancement](https://github.com/vxcontrol/soldr/blob/master/.github/ISSUE_TEMPLATE/2-enhancement.md), and [question](https://github.com/vxcontrol/soldr/blob/master/.github/ISSUE_TEMPLATE/3-help.md) requests.  I would appreciate it if you could edit your Issue and add in the missing details.
```

Comment:
Link to "3-help.md" will be broken as this template was not added by this PR. Can you, please, add it to fill a gap?

Reply:
Template removed, because now github have "Discussions" functionality